### DISCUSSION TITLE
طراحی طرح های آماده پوستر و تی شرت برای دانلود رایگان و شرکت در تظاهرات

### DISCUSSION THREAD

**Discussion by Bitashf** [WeAreMahsaAmini/Core#327] [Ideas]
طراحی طرح های آماده پوستر و تی شرت برای دانلود رایگان و شرکت در تظاهرات

**Comment by SamanZargarzadeh** [comment]
سلام
فکر کنم بتونیم این ایده شما رو با #428 ترکیب کنیم من توی طراحی سایت سررشته‌ای ندارم ولی برای جمع‌آوری طرح‌ها و درست کردن آرشیو می‌تونم کمک کنم. یکم هم فوتوشاپ بلدم که برای میکس کردن طراحی‌ها واسه چاپ تیشرت و پلاکارد شاید به درد بخوره.
